blocks/header-nav-chain-block/chains/header-nav-chain-block/_children/horizontal-links/default.jsx

```diff
@@ -44,31 +36,35 @@ const HorizontalLinksBar = ({ hierarchy, navBarColor, showHorizontalSeperatorDot
 		content &&
 		content.children &&
 		content.children.length > 1 &&
-		showHorizontalSeperatorDots
+		showHorizontalSeparatorDots
 	);
 
-	// the interior links and spans need to contrast with nav color scheme
-	const readableContrastingColor = navBarColor === "light" ? "#000" : "#fff";
+	// don't show menu items container if there are no menu items
+	if (menuItems.length === 0) {
+		return null;
+	}
 
 	return (
-		<ReadableTextNavigationBar
+		<Stack
 			key={id}
-			color={readableContrastingColor}
-			className="horizontal-links-bar"
+			as="nav"
+			direction="horizontal"
+			className="b-header-nav-chain__links-list"
```

Comment:
```suggestion
			className=`${BLOCK_CLASS_NAME}__links-list`
```
May have this already on your list, just wanted to make a trackable note.

Could even import it from the `default.jsx` so it's consistent if changed later.

Reply:
addressed